packages/react-static-plugin-preact/README.md

```diff
@@ -16,4 +16,28 @@ Then add the plugin to your `static.config.js`:
 export default {
   plugins: ["react-static-plugin-preact"]
 };
-```
\ No newline at end of file
+```
+
+As final step, depending on the template you started with, you might need to remove the React Hot Reload component from `index.js`. Change:
+
+```
+  const render = Comp => {
+    renderMethod(
+      <AppContainer>
+        <Comp />
+      </AppContainer>,
+      target
+    )
+  }
+```
+
+to:
+
+```
+  const render = Comp => {
+    renderMethod(
+      <Comp />,
+      target
+    )
+  }
+```
```

Comment:
is there a way to apply this in a more automated way when the plugin is installed?

Reply:
Yes, we could provide a `codemod`, or run a postinstall script.